stravalib/model.py

```diff
@@ -638,6 +638,7 @@ class Segment(
     map: Optional[Map] = None
     athlete_segment_stats: Optional[AthleteSegmentStats] = None
     athlete_pr_effort: Optional[AthletePrEffort] = None
+    activity_type = Optional[ActivityType] = None
```

Comment:
```suggestion
    activity_type: Optional[ActivityType] = None
```

Can you please correct this to make the unit tests succeed? Thanks!

Reply:
Wait, I can do this myself 🙈